module/userProjects/helper.js

```diff
@@ -2987,6 +3152,119 @@ module.exports = class UserProjectsHelper {
 			}
 		})
 	}
+
+	/**
+	 * Verify project certificate
+	 * @method
+	 * @name verifyCertificate
+	 * @param {String} projectId - projectId.
+	 * @returns {JSON} certificate details.
+	 */
+
+	static verifyCertificate(projectId) {
```

Comment:
@Prajwal17Tunerlabs add validation for this function

Reply:
validation added to verifyCertificate api.